gfw_pixetl/pixetl_prep.py

```diff
@@ -26,16 +29,27 @@ def create_geojsons(
 
     tiles: List[DummyTile] = list()
 
+    # Assume the geojson files have been created correctly already, if an
+    # even number already exist.  This is to allow progress when there are a
+    # very large number of tiles, typically zoom_14 (see GTC-3035).
+    data_lake_bucket = get_bucket()
+    target_prefix = f"{dataset}/{version}/{prefix.strip('/')}/"
+    geojsons = get_aws_files(data_lake_bucket, target_prefix, (".geojson"))
+    if len(geojsons) >= 2 and len(geojsons) % 2 == 0:
+        LOGGER.info(f"tiles.geojson and extent.geojson already exist, skipping creation of geojsons:\n{geojsons}")
+        return
+
     for provider, bucket, key in resources:
         files = get_files[provider](bucket, key)
+        count = 0
+        total = len(files)
 
         for uri in files:
+            LOGGER.info(f"Reading tile {count} of {total}")
+            count += 1
```

Comment:
BTW, this is slightly more idiomatically done with 
for i, uri in enumerate(files):
    LOGGER.info(f"Reading tile {i} of {total}")
    ...
But this works.